yaml_test.go

```diff
@@ -165,7 +165,6 @@ func prettyFunctionName(opts []JSONOpt) []string {
 }
 
 func unmarshalEqual(t *testing.T, y []byte, s, e interface{}, opts ...JSONOpt) {
-	t.Helper()
```

Comment:
Don't remove this.

We don't care about ancient Go versions.

Reply:
Unfortunately, this little `t.Helper()` would break Travis and Shippable. I added it in this PR and removed it again for compatibility. Can we move the discussion about `t.Helper()`, ancient Go versions, and the integration test setup to a separate PR?

Reply:
WDYT?

Reply:
Ping :smile:

Reply:
I haven't heard back from @ghodss so I'll make the policy decision: we no longer care about Go 1.3 and Go 1.4.

In fact, I think it'd be fine to move our requirement up to Go 1.9 (when Helper was introduced).

So why don't you make those changes, including to the yaml file. You can make that first in a separate PR actually.

Reply:
Thx/